common/persistence/sql/sqlVisibilityStore.go

```diff
@@ -250,7 +250,7 @@ func (s *sqlVisibilityStore) GetClosedWorkflowExecution(
 	rows, err := s.db.SelectFromVisibility(ctx, &sqlplugin.VisibilityFilter{
 		DomainID: request.DomainUUID,
 		Closed:   true,
-		RunID:    execution.RunID,
+		RunID:    &execution.RunID,
```

Comment:
I'll resolve this if it turns out safe as I read more: this has a semantic change from allowing nulls to pass through -> no more nulls.  does that matter?

Reply:
ah.  this one's fine, it'd be useless to not pass at least a runid.  great.

on further reading though: maybe this func's args are just too imprecise? it accepts but ignores domain and workflow-id, and... _overwrites_ the workflowid on the returned data with whatever was passed in?  that's weird looking.